requirements.txt

```diff
@@ -9,3 +9,4 @@ ont-fast5-api==3.0.1
 fast-ctc-decode==0.2.5
 bonito-cuda-runtime==0.0.2a2
 #pyclaragenomics-cuda-10-0==0.4.2
+openvino-python==2021.1
```

Comment:
@dkurt I am supporting Ubuntu 16.04 and 20.04 so only having wheels for 3.6 and 3.7 is too limiting. Can you see about getting wheels published for Python 3.5 and 3.8?

Reply:
Hi!

There are plans to add Ubuntu 20.04 wheels. Not sure about Ubuntu 16.04 right now.

Reply:
Is there an ETA on 3.8 wheels on PyPI?

Reply:
At this moment I can only recommend to install OpenVINO manually for unsupported configurations:
```bash
sudo curl -o GPG-PUB-KEY-INTEL-OPENVINO-2021 https://apt.repos.intel.com/openvino/2021/GPG-PUB-KEY-INTEL-OPENVINO-2021
sudo apt-key add GPG-PUB-KEY-INTEL-OPENVINO-2021
sudo echo "deb https://apt.repos.intel.com/openvino/2021 all main" | sudo tee - a /etc/apt/sources.list.d/intel-openvino-2021.list
sudo apt-get update
```
* Ubuntu 20.04
```bash
sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu20-2021.1.110
```
* Ubuntu 18.04
```bash
sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu18-2021.1.110
```

Reply:
fixed